src/compatTest/groovy/io/github/cosmicsilence/scalafix/ScalafixPluginFunctionalTest.groovy

```diff
@@ -3,50 +3,19 @@ package io.github.cosmicsilence.scalafix
 import org.gradle.testkit.runner.BuildResult
 import org.gradle.testkit.runner.GradleRunner
 import org.gradle.testkit.runner.UnexpectedBuildFailure
-import org.junit.Rule
 import org.junit.rules.TemporaryFolder
 
 import spock.lang.Specification
+import spock.lang.Unroll
 
 class ScalafixPluginFunctionalTest extends Specification {
```

Comment:
I had to do a bit of refactoring here to be able to run tests using different versions of Scala